test/integration/disable_client_cert/controls/gcloud.rb

```diff
@@ -17,7 +17,6 @@
 cluster_name = attribute('cluster_name')
 
 credentials_path = attribute('credentials_path')
```

Comment:
This attribute and the associated Terraform output should be removed.

Reply:
After disable cluster_name = attribute('cluster_name') I have a problem with kitchen verify, gktase take a look:
`kitchen verify disable-client-cert-local`
-----> Starting Kitchen (v1.24.0)
WARN: Unresolved specs during Gem::Specification.reset:
      azure_mgmt_resources (~> 0.15)
      ms_rest_azure (~> 0.11.0)
      public_suffix (< 4.0, >= 2.0.2)
      jwt (< 3.0, >= 1.4, >= 1.5)
WARN: Clearing out unresolved specs.
Please report a bug if this causes problems.
-----> Verifying <disable-client-cert-local>...
$$$$$$ Running command `terraform workspace select kitchen-terraform-disable-client-cert-local` in directory /home/weannye/work/gcp/forks/terraform-google-kubernetes-engine/test/fixtures/disable_client_cert
$$$$$$ Running command `terraform output -json` in directory /home/weannye/work/gcp/forks/terraform-google-kubernetes-engine/test/fixtures/disable_client_cert
disable_client_cert: Verifying

Profile: disable_client_cert
Version: (not specified)
Target:  local://

  ×  gcloud: Google Compute Engine GKE configuration
     ×  Control Source Code Error /home/weannye/work/gcp/forks/terraform-google-kubernetes-engine/test/integration/disable_client_cert/controls/gcloud.rb:21 
     undefined local variable or method `cluster_name' for #<#<Class:0x0000000005f91b68>:0x0000000005f8e1e8>


Profile Summary: 0 successful controls, 1 control failure, 0 controls skipped
Test Summary: 0 successful, 1 failure, 0 skipped
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: 1 actions failed.
>>>>>>     Verify failed on instance <disable-client-cert-local>.  Please see .kitchen/logs/disable-client-cert-local.log for more details
>>>>>> ----------------------
>>>>>> Please see .kitchen/logs/kitchen.log for more details
>>>>>> Also try running `kitchen diagnose --all` for configuration

Reply:
@ingwarr my comment was related to `credentials_path`, not `cluster_name`.

Reply:
Oops, sorry, it was not obvious for me

Reply:
Done